src/main/java/sirius/biz/importer/BaseImportHandler.java

```diff
@@ -70,6 +75,8 @@
     @Parts(EntityImportHandlerExtender.class)
     protected static PartCollection<EntityImportHandlerExtender> extenders;
 
+    private static final int DEFAULT_MESSAGE_TYPE_COUNT = 250;
```

Comment:
declare one or two constants in ProcesLog? we have a bunch of magic numbers throughout the code
 - may be we define 100 and 250 as two common limit (?)